### DISCUSSION TITLE
How to map delete key?

### DISCUSSION THREAD

**Discussion by txtyash** [fish-shell/fish-shell#8709] [Q&A] [answered]
I tried a lot to check if remapping keys like end, insert, delete works but every binding like such:
```
            bind -M $mode end kill-whole-line repaint
```
failed.

I wanted to map the delete key to delete a word like Ctrl+w

BTW, The list of keys outputted by `bind -K` is vague. The key names aren't descriptive. At least add a reference page online where I can read what sic, sf, sr, beg, etc. mean.

**Comment by krobelus** [comment, marked as answer]
On Tue, Feb 08, 2022 at 08:45:05PM -0800, zim0369 wrote:
> I tried a lot to check if remapping keys like end, insert, delete works but every binding like such:
> ```
>             bind -M $mode end kill-whole-line repaint
> ```
> failed.

use `fish_key_reader` to find out the name of your key
